app/models/alert.py

```diff
@@ -9,7 +9,6 @@
 class BaseAlert(SerialisedModel):
     ALLOWED_PROPERTIES = {
         'starts_at',
-        'content',
```

Comment:
How come the field is diverging between the two subclasses now? It would be simpler to use the same field name for both kinds of alert, especially since they're pretty much synonyms of each other - I'm not sure what the difference is.

Reply:
`content` is the message the user will get on their phone. `description` is information _about_ the alert, for example when it will happen and who will receive it. I’m not wedded to that naming though, or the particular class structure if you can think of a clearer way of expressing it.

For context here is where these things appear, and how they are shown differs from one place to another:

— | Real alert | Operator alert | Upcoming test 
----|-----------|----------------|-------------- 
**Instance** | `Alert` | `Alert` | `PlannedTest`
**‘Past alerts’ page** | `content` shown | `content` shown | Not shown
**‘Planned tests’ page** | Not shown | Generic text shown | `description` or generic text shown

Reply:
Aha, I see where you're coming from now. The field name makes sense with your explanation, but what's confusing it for me is the class hierarchy as this brings it into conflict with `content`.

Looking through the code, we don't seem to be using many of the normal alert fields for planned tests anyway. Arguably a planned test isn't an `Alert`, either: it may actually represent many alerts. 

I'm wondering how hard it would be to just make `PlannedTest` a child of `Object` and copy across the small amount of functionality it shares with `Alert`. Then we get more flexibility in future too.

What do you think?

Reply:
I’ve tried that in 16557aa and think it’s an improvement. Means we get to keep the pattern of having on class per file as well, which is nice.